### PR TITLE
Updating the limitaions on using complex file formats

### PR DESCRIPTION
**Pull request by chitara-01**
<h4> Summary (Short summary of what is being done) : </h4>
Updating the limitaions on using complex file formats
<br>
<h4> Description (Describe in detail the fix made) : </h4>
Based on the recent debugging and research done on AVRO/Parquet/JSONL pipelines, there are some limitations in the existing implementation. This pull request updates the README guide to make users aware of those limitations. 
<br>
<h4> Bug ID (if any) : </h4>
b/310711896
<br>
<h4> Public Documentation (if any) : </h4>
<br>
<h4> TESTED (Test Cases with scenario and description - must have 1 positive and 1 negative scenario) : </h4>
Not required.

### CODE REVIEW

**Review by dup05 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -506,6 +506,10 @@ To run the pipeline for JSONL files, the list of comma-separated headers also ne
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/CCRecords_sample.jsonl --headers=<comma_separated_list_of_headers>"
 ```
 
+Limitaions:
+1. The original schema of the input file will not be preserved as the data is simplified while converting it to DLP API
```
Comment (catherinerosesantos) [inline README.md:510]: Since there's only one limitation, we don't need the 1.

File: README.md [hunk 2]
```diff
@@ -506,6 +506,10 @@ To run the pipeline for JSONL files, the list of comma-separated headers also ne
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/CCRecords_sample.jsonl --headers=<comma_separated_list_of_headers>"
 ```
 
+Limitaions:
```
Comment (catherinerosesantos) [inline README.md:509]: Fix the typo or just remove this heading.

File: README.md (hunk 1 above)
Comment (catherinerosesantos) [reply README.md:510]: How about:

The original schema of the input file will not be preserved. This solution simplifies the data when it converts the data to a DLP API request object.

File: README.md [hunk 3]
```diff
@@ -515,6 +519,10 @@ the pipeline except updating the `--filePattern` parameter. For example:
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.avro"
 ```
 
+Limitaions:
```
Comment (catherinerosesantos) [inline README.md:522]: Fix the typo or just remove this heading.

File: README.md [hunk 4]
```diff
@@ -515,6 +519,10 @@ the pipeline except updating the `--filePattern` parameter. For example:
...
+1. The original schema of the input file will not be preserved as the data is simplified while converting it to DLP API
```
Comment (catherinerosesantos) [inline README.md:523]: Same suggestion as above.

File: README.md [hunk 5]
```diff
@@ -535,6 +543,10 @@ No additional changes are required to run the pipeline except updating the `--fi
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.parquet"
 ```
 
+Limitaions:
+1. The original schema of the input file will not be preserved as the data is simplified while converting it to DLP API
```
Comment (catherinerosesantos) [inline README.md:547]: Same suggestion as above.

File: README.md [hunk 6]
```diff
@@ -535,6 +543,10 @@ No additional changes are required to run the pipeline except updating the `--fi
 ./gradlew run ... -Pargs="... --filePattern=gs://${PROJECT_ID}-demo-data/*.parquet"
 ```
 
+Limitaions:
```
Comment (catherinerosesantos) [inline README.md:546]: Fix the typo or just remove this heading.

File: README.md [hunk 7]
```diff
@@ -562,10 +574,12 @@ written in a BigQuery dataset as tables or an output Cloud Storage bucket as ORC
 --outputBucket=<output_storage_bucket> ..."
 ```
 
-In the above command, replace <i>output_storage_bucket</i> with the URI of the Cloud Storage bucket where you want to store the 
-de-identified ORC files.
+In the above command, replace <i>output_storage_bucket</i> with the URI of the Cloud Storage bucket where you want to 
+store the de-identified ORC files. The de-identification pipeline supports input files with varying schemas to be
```
Comment (catherinerosesantos) [inline README.md:578]: supports --> allows

File: README.md (hunk 5 above)
Comment (catherinerosesantos) [reply README.md:547]: Please apply these changes to the Parquet step too (including the typo fix)

File: README.md [hunk 8]
```diff
@@ -562,10 +574,12 @@ written in a BigQuery dataset as tables or an output Cloud Storage bucket as ORC
...
 
-In the above command, replace <i>output_storage_bucket</i> with the URI of the Cloud Storage bucket where you want to store the 
-de-identified ORC files.
+In the above command, replace <i>output_storage_bucket</i> with the URI of the Cloud Storage bucket where you want to 
+store the de-identified ORC files. The de-identification pipeline supports input files with varying schemas to be 
+processed in the same pipeline.
 
-The de-identification pipeline supports input files with varying schemas. Currently, it can process only primitive
+Limitaions:
+1. Currently, it can process only primitive
```
Comment (catherinerosesantos) [inline README.md:582]: it --> this solution

**Review by catherinerosesantos — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chitara-01) [reply README.md]: Thanks @catherinerosesantos for all the corrections. I have pushed the new changes. Please review.

**Review by catherinerosesantos — APPROVED** [review]

**Pull request merged**